.goreleaser.yml

```diff
@@ -47,6 +47,10 @@ nfpms:
       - src: package-files/connector.example.yaml
         dst: /etc/infra/connector.example.yaml
         type: config
+      - src: package-files/rpm/infra.pp
```

Comment:
Do we track the source code of this file anywhere? How do we update it?

Reply:
How difficult would it be to include the source in git, and build this `.pp` as part of the release?  We don't need goreleaser to build it, we can do it as a step before goreleaser.

Reply:
We could but it's pretty annoying. It requires `checkmodule` to compile `*.te` into a `*.mod` then `semodule_package` to package `*.mod` into a `*.pp`. Each binary comes from a separate package, `checkpolicy` for `checkmodule` and `policycoreutils` for `semodule_package`, on Fedora 36. I would rather just turn on `nis_enabled` in that case

Reply:
Installing a couple packages doesn't seem like a big problem. What am I missing? Are these packages particularly large and slow to download?